docs/LocalDeployment.md

```diff
@@ -0,0 +1,197 @@
+
+This tutorial will cover how to use Juju and Charmed Operators to deploy an instance of the [FINOS Waltz](https://waltz.finos.org/) charm on your local workstation, using [Microk8s](https://microk8s.io/). It can be used for evaluation and demonstration purposes, as well as development of new Waltz features.
+
+These steps may be carried out on any Linux distribution that has a [Snap](https://snapcraft.io/) package manager installed. The steps in this tutorial have been tested on [Ubuntu 20.04](https://releases.ubuntu.com/focal/).
+
+The deployment is composed by the following steps:
+
+- [Install Microk8s](#Install-Microk8s)
+- [Install Juju](#Install-Juju)
+- [Bootstrap Juju](#Bootstrap-Juju)
+- [Add the Juju Waltz Model](#Add-the-Juju-Waltz-Model)
+- [Build and deploy the Waltz Charm](#Build-and-deploy-the-Waltz-Charm)
+- [Monitor Juju status](#Monitor-Juju-status)
+- [Create a PostgreSQL database](#Create-a-PostgreSQL-database)
+- [Use Waltz](#Use-Waltz)
+- [Updating the Waltz Charm](#Updating-the-Waltz-charm) (optional)
+- [Destroy setup](#Destroy-setup) (optional)
+
+## Install Microk8s
+
+[Microk8s](https://microk8s.io/) is a *micro* Kubernetes distribution that runs locally; you can install it on Linux running the following commands:
+
+```bash
+sudo snap install microk8s --classic
+sudo snap alias microk8s.kubectl kubectl
+source ~/.profile
+sudo usermod -a -G microk8s $USER
+sudo chown -f -R $USER ~/.kube
+newgrp microk8s
+```
+
+To configure and start `microk8s`, simply run:
+
+```bash
+microk8s enable dns storage
+microk8s status --wait-ready
+```
+
+Below is the expected output; before moving forward, wait to see that `dns` and `storage` are listed as `enabled`.
+
+```bash
+microk8s is running
+high-availability: no
+  datastore main nodes: 127.0.0.1:19001
+  datastore standby nodes: none
+addons:
+  enabled:
+    dns                  # CoreDNS
+    ha-cluster           # Configure high availability on the current node
+    storage              # Storage class; allocates storage from host directory
+...
+```
+
+## Install Juju
+
+To install Juju, you can follow [the instructions in the docs](https://juju.is/docs/olm/installing-juju) or simply install a Juju with the command line `sudo snap install juju --classic`. Run `juju status` to check if everything is up.
+
+If you're interested to know how to run Juju on your cloud of choice, checkout [the official docs](https://juju.is/docs/olm/clouds); you can always run `juju clouds` to check your configured clouds. In the instructions below, we will use `microk8s`, but you can replace it with the name of the cloud you're using.
+
+## Bootstrap Juju
+
+In Juju terms, "bootstrap" means "create a Juju controller", which is the part of Juju that runs in your cluster and controls the applications.
+
+```bash
+juju bootstrap microk8s waltz-controller
+```
+
+You are free to use any name other than `waltz-controller` but it must be consistent through the rest of this tutorial.
```

Comment:
Might be better to just point at: https://juju.is/docs/sdk/dev-setup

That way we only have one dev setup doc to maintain? :)

Reply:
Done